AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/ActivityAdminController.cs

```diff
@@ -57,17 +57,31 @@ public override ViewResult View(string viewName, object model)
         }
 
         // GET: Activity
-        public async Task<IActionResult> Index()
+        [Route("Admin/Activity/{campaignId}")]
+        public IActionResult Index(int campaignId)
         {
-            return await Task.Run(() => View(_dataAccess.Activities));
+            Campaign campaign = _dataAccess.GetCampaign(campaignId);
+            if (!UserIsTenantAdmin(campaign.ManagingTenantId))
+            {
+                return HttpUnauthorized();
+            }
+
+            ViewBag.Title = "Activities for " + campaign.Name;
```

Comment:
Since it's a ViewBag, it's essentially setting it on the view. I think this is a good opportunity to propose setting up a base ViewModel that has standard properties for things like Title, Subtitle, etc. It may just be a matter of personal preference, but I've never been a big fan of the ViewBag, due to its lack of type safety and lack of discoverability.